docs/DeveloperGuide.md

```diff
@@ -692,9 +693,13 @@ Expected: An error message indicating that the show was not found.
 
 Expected: An error message displayed due to the different number of seasons and episodes for each season given.
 
-3. Test case: ``
+3. Test case: `duration 1h0m` followed by `done`
 
-4. Test case: ``
+Expected: A message acknowledging the change. `Updated show details.`
+
+4. Test case: `season 3` followed by `episode 10,10,10`
```

Comment:
Not sure but I think need to include the 'done' command also for test case

Reply:
Ok, on it